org-formation/organization.yaml

```diff
@@ -39,19 +38,22 @@ Organization:
         - !Ref ScicompAccount
         - !Ref iAtlasProdAccount
         - !Ref WorkflowsNextflowProdAccount
+#              StridesPoolAccount must be managed outside of orgf but should have parity with this OU
 
   PlatformOU:
```

Comment:
do we need PlatformOU now that there's nothing in it?

Reply:
I left the PlatformOU in to contain the SynapseProdOU and BridgeProdOU ... that makes sense, right?

Reply:
oops, sorry.  i missed that.